src/main/java/edu/harvard/iq/dataverse/util/SignpostingResources.java

```diff
@@ -63,46 +56,47 @@ public SignpostingResources(SystemConfig systemConfig, DatasetVersion workingDat
         licJsonObj = spJsonSetting.getJsonObject("license");
-        citeAsJsonObj = spJsonSetting.getJsonObject("cite-as");
+        describedByJsonObj = spJsonSetting.getJsonObject("describedby");
         useDefaultFileType = spJsonSetting.getBoolean("useDefaultFileType", true);
         defaultFileTypeValue = spJsonSetting.getString("defaultFileTypeValue", "https://schema.org/Dataset");
+        maxAuthors = spJsonSetting.getInt("maxAuthors", 5);
+        maxItems = spJsonSetting.getInt("maxItems", 5);
     }
 
     /**
      * Get identifier schema for each author
      *
-     * Author may have identifiers from different providers
-     * ORCID: the url format is https://orcid.org/:id
-     * ISNI: the url format is https://isni.org/isni/:id
-     * ScopusID: the url format is https://www.scopus.com/authid/detail.uri?authorId=:id
-     * VIAF: the url format is http://viaf.org/viaf/:id
-     *
      * For example:
-     *      if author has VIAF
-    *       Link: <http://viaf.org/viaf/:id/>; rel="author"
+     * if author has VIAF
+     * Link: <http://viaf.org/viaf/:id/>; rel="author"
      *
-     * @param datasetAuthors
-     * @return
+     * @param datasetAuthors list of all DatasetAuthor object
+     * @return all the non empty author links in a string
      */
     private String getIdentifierSchema(List<DatasetAuthor> datasetAuthors) {
-        StringBuilder sb = new StringBuilder();
+        String singleAuthorString;
         String identifierSchema = "";
 
-        for (DatasetAuthor da:datasetAuthors){
-            if (da.getIdValue() != null && !da.getIdValue().isEmpty()) {
-                sb.append("<").
-                        append(idschemaJsonObj.getString(da.getIdType())).
-                        append(da.getIdValue()).
-                        append(">;rel=\"author\"");
-                if (identifierSchema == "") {
-                    identifierSchema = sb.toString();
+        for (DatasetAuthor da : datasetAuthors) {
+            logger.info(String.format(
+                    "idtype: %s; idvalue: %s, affiliation: %s; identifierUrl: %s",
+                    da.getIdType(),
+                    da.getIdValue(),
+                    da.getAffiliation(),
+                    da.getIdentifierAsUrl()
+            ));
+            if (da.getIdentifierAsUrl() != null && !da.getIdentifierAsUrl().trim().isEmpty()) {
+                singleAuthorString = "<" + da.getIdentifierAsUrl() + ">;rel=\"author\"";
+                if (Objects.equals(identifierSchema, "")) {
+                    identifierSchema = singleAuthorString;
                 } else {
-                    identifierSchema = String.join(",", identifierSchema, sb.toString());
+                    identifierSchema = String.join(",", identifierSchema, singleAuthorString);
                 }
             }
         }
+
+        logger.info(String.format("identifierSchema: %s", identifierSchema));
```

Comment:
Does it make sense to have a log statement here? If so, it should be more informative and at debug level.